analysis_options.yaml

```
@@ -1,8 +1,37 @@
-include: package:effective_dart/analysis_options.1.3.2.yaml
+# This file configures the analyzer, which statically analyzes Dart code to
+# check for errors, warnings, and lints.
+#
+# The issues identified by the analyzer are surfaced in the UI of Dart-enabled
+# IDEs (https://dart.dev/tools#ides-and-editors). The analyzer can also be
+# invoked from the command line by running `flutter analyze`.
+
+# The following line activates a set of recommended lints for Flutter apps,
+# packages, and plugins designed to encourage good coding practices.
+include: package:flutter_lints/flutter.yaml
 
 linter:
-    rules:
-        constant_identifier_names: false
-        omit_local_variable_types: false
-        avoid_types_on_closure_parameters: false
-        use_setters_to_change_properties: false
+  # The lint rules applied to this project can be customized in the
+  # section below to disable rules from the `package:flutter_lints/flutter.yaml`
+  # included above or to enable additional rules. A list of all available lints
+  # and their documentation is published at
+  # https://dart-lang.github.io/linter/lints/index.html.
+  #
+  # Instead of disabling a lint rule for the entire project in the
+  # section below, it can also be suppressed for a single line of code
+  # or a specific dart file by using the `// ignore: name_of_lint` and
+  # `// ignore_for_file: name_of_lint` syntax on the line or in the file
+  # producing the lint.
+  rules:
+    implementation_imports: false
```

please add back the removed rules
```suggestion
    implementation_imports: false
    constant_identifier_names: false
    omit_local_variable_types: false
    avoid_types_on_closure_parameters: false
    use_setters_to_change_properties: false
```